source/PlayerInfo.cpp

```diff
@@ -4009,6 +4009,7 @@ void PlayerInfo::CreateMissions()
 	// Check for available missions.
 	bool skipJobs = planet && !planet->GetPort().HasService(Port::ServicesType::JobBoard);
 	bool hasPriorityMissions = false;
+	unsigned nonBlockingMissions = 0;
```

Comment:
I'd expect non-blocking missions to just always be offered (regardless of anything else), and not blocking anything else. So don't let them run in parallel with minor missions, but just let them run in parallel with everything (or in this code, run them ahead of any other mission).

Reply:
As discussed on discord; this code is indeed correct and needed to ensure that minor missions still offer.